### DISCUSSION TITLE
Drawing spline shapes ?

### DISCUSSION THREAD

**Discussion by nyue** [cduck/drawSvg#64] [Q&A] [answered]
Hi,

  I have been able to draw shapes with straight lines using Path.

  I looked at some SVG file with spline and came up with the following, is this the correct way to draw spline shape ?

```
import drawSvg as draw

d = draw.Drawing(400, 400, origin='center', displayInline=False)

p1 = draw.Path(stroke='green', stroke_width=2, fill='green')
p1.m(31.75,42.244047).c(0,0, 68.055716,12.241243, 97.51786,35.529762).c(30.05693,22.497401,25.70238,84.666671, 25.70238,84.666671)
d.append(p1)

d.saveSvg('test.svg')
d.savePng('test.png')
```

Cheers

**Comment by cduck** [comment, marked as answer]
There are multiple types of curves.  MDN is a great reference for all SVG features.  https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths should explain everything you need.
